CONTRIBUTING.md

```diff
@@ -60,6 +62,34 @@ The `rustup::currentprocess` module abstracts the global state that is
 `std::env::current_dir` and `std::process::exit` permitting threaded tests of
 the CLI logic; use `process()` rather than those APIs directly.
 
+### Clippy lints
+
+We don't enforce lint status in the checks done by GitHub Actions, because
```

Comment:
```suggestion
We do not enforce lint status in the checks done by GitHub Actions, because
```
`don't` is more useful on normal conversation, not talking.